style/app.scss

```diff
@@ -642,186 +642,168 @@ a:active
 }
 
-#reset
+.expanded > #sidebar > #sidebar-button > #sidebar-icon
 {
-  text-align : center;
-  float      : left;
-  width      : 47%;
-  height     : 40px;
-  border     : solid 1px #000000;
-  cursor     : pointer;
+  -webkit-transform : rotate(180deg);
+  -moz-transform    : rotate(180deg);
+  -ms-transform     : rotate(180deg);
+  -o-transform      : rotate(180deg);
+  transform         : rotate(180deg);
 }
 
-#reset:hover
+.collapsed > #sidebar
 {
-  background-color : #222222;
-  color            : #ffffff;
+  height : 0px;
+  width  : 0px;
 }
 
-#container
+.collapsed > #sidebar > #selected-courses
 {
-  height   : calc(100% - 50px);
-  width    : 100%;
-  position : relative;
-  background: #f0f5f8;
+  height : 0px;
+  width  : 0px;
 }
 
-#sidebar
+.collapsed > #sidebar > .courses
 {
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #7a6a96;
-  position         : absolute;
-  padding-left     : 23px;
+  height : 0px;
+  width  : 0px;
 }
 
-#sidebar-button
+.collapsed > #sidebar > #reset-selections
 {
-  cursor           : pointer;
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #5c497e;
-  position         : absolute;
-  border           : solid 1px #000000;
+  height : 0px;
+  width  : 0px;
+  margin : 0px;
 }
 
-#sidebar-button:hover
+.collapsed > #sidebar > #sidebar-button
 {
-  background-color : #cac4d4;
+  margin-top : -25px;
 }
 
-#sidebar-icon
+.expanded > #sidebar > #sidebar-button
 {
-  width        : 30px;
-  height       : 50px;
-  padding-top  : 20px;
-  padding-left : 2px;
-  position     : absolute;
-  top          : 40%;
-  left         : 4px;
+  margin-top : -5px;
 }
 
-#focuses-nav, #graph-nav
-{
-  cursor : pointer;
-}
-
-#sidebar-nav
+#sidebar-button
 {
-  width            : 100%;
-  font-size        : 13px;
-  background-color : #cac4d4;
-  border           : solid 1px #dedede;
-  box-shadow       : 0 2px 2px -1px rgba(0, 0, 0, 0.055);
-  display          : block;
-  overflow         : hidden;
+  cursor           : pointer;
+  border-radius    : 10px;
+  width            : 30px;
+  height           : 15px;
+  background-color : #d9d9d9;
+  margin-left      : 60px;
+  text-align       : center;
+  visibility       : visible !important;
 }
 
-#sidebar-nav ul
+#selected-courses
 {
-  width  : 100%;
-  margin : 0 0 0 0;
+  margin-top       : 10px;
+  margin-bottom    : 10px;
+  font-size        : small;
+  color            : #a47cb8;
 }
 
-#sidebar-nav ul li
+.courses
 {
-  list-style-type    : none;
-  display            : inline-block;
-  width              : 48%;
-  -webkit-transition : all 0.2s;
-  -moz-transition    : all 0.2s;
-  -ms-transition     : all 0.2s;
-  -o-transition      : all 0.2s;
-  transition         : all 0.2s;
+  overflow         : auto;
+  height           : 275px;
+  width            : 105px;
```

Comment:
You shouldn't be setting a separate width for any of the children. You can add `padding` to the parent element.